R/heatmap_mutation_frequency_bin.R

```diff
@@ -33,6 +33,7 @@
 #' @param cluster_cols_heatmap Boolean.  Default FALSE.
 #' @param show_gene_colours Boolean. Whether to add heatmap annotation colours for each region. Default FALSE.
 #' @param label_regions_by Specify which feature of the regions to label the heatmap with. Heatmap will be split according to this value, and ordered by factor levels if the specified column is a factor. Default name.
+#' @param label_regions_rotate Specify degree by which the label in the previous parameter will be rotated. Default 0 (no rotation).
```

Comment:
Need to include that only 0, 90 and 270 values are possible ([ComplexHeatmap::Heatmap](https://www.rdocumentation.org/packages/ComplexHeatmap/versions/1.10.2/topics/Heatmap)).